### PR TITLE
Subscription form

### PR DESCRIPTION
**Pull request by AlexisFLT**

### CODE REVIEW

**Review by Sax593 — DISMISSED** [review]
It's ok

File: frontend/src/App.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+$mainHeight: 100vh;
+$color-background:#70af90;
+$color-primary: #196c84;
+$text-color-primary: #0c3944;
+$text-color-secondary: #fff30d;
+$font-text: Helvetica, Arial, sans-serif;
+
+
```
Comment (LoicBrassart) [inline frontend/src/App.scss]: Prettier should have fixed this ಠ_ಠ

File: frontend/src/pages/SubscriptionForm/style.scss [hunk 2]
```diff
@@ -0,0 +1,123 @@
+@use '../../App';
+
+
```
Comment (LoicBrassart) [inline frontend/src/pages/SubscriptionForm/style.scss]: Give me back my Prettier! ಠ_ಥ

File: frontend/src/pages/SubscriptionForm/style.scss [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    border: 1px #e0e4eb solid;
+    padding: 1em;
+    border-radius: 1em;
+    box-shadow: 0 20px 60px 0 black;
+    background-color: #ffffff;
+    margin-top: 3%;
+
+}
+
+
```
Comment (LoicBrassart) [inline frontend/src/pages/SubscriptionForm/style.scss:28]: ...

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Fix your Prettier, damnit! ୧( ಠ Д ಠ )୨

**Review by Sax593 — DISMISSED** [review]
It's OK

File: frontend/package.json [hunk 4]
```diff
@@ -11,6 +11,9 @@
   },
   "dependencies": {
     "node-sass": "^8.0.0",
+
+    "@makesenseorg/design-system": "^1.16.3",
+
```
Comment (Sax593) [inline frontend/package.json]: Unnecessary space

File: frontend/src/App.jsx [hunk 5]
```diff
@@ -8,6 +8,7 @@ function App() {
 
     </div>
   );
+
```
Comment (Sax593) [inline frontend/src/App.jsx]: Unnecessary space

**Review by Sax593 — CHANGES_REQUESTED** [review]
Unnecessary space

**Review by Sax593 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**